cmd/tfengine/main.go

```diff
@@ -49,7 +51,19 @@ func main() {
 	}
 	defer os.RemoveAll(cacheDir)
 
-	opts := &tfengine.Options{Format: *format, CacheDir: cacheDir}
+	templatesMap := make(map[string]bool)
+	for _, t := range strings.Split(*templates, ",") {
+		t = strings.TrimSpace(t)
+		if len(t) > 0 {
```

Comment:
do we need this if we have trimmed spaces?

Reply:
Yes. Otherwise, if you provide no flag or `--templates ,,`, it ends up mapping `""` (empty string) to `true`.